src/java/picard/illumina/quality/CollectHiSeqXPfFailMetrics.java

```diff
@@ -58,12 +58,65 @@
+            "following four categories: " +
+            "<ul>" +
+            "<li>MISALIGNED - The first 24 basecalls of a read are uncalled (numNs~24).  " +
+            " These types of reads appear to be flow cell artifacts because reads " +
+            "were only found near tile boundaries and were concentration (library) independent</li> " +
+            "<li>Empty wells - All 24 bases are called (numNs~0) but the number of bases with quality scores" +
+            " greater than two is less than or equal to eight (numQGtTwo<=8).  These reads were location independent within the tiles and " +
+            "were inversely proportional to the library concentration</li>" +
+            "" +
+            "<li>Polyclonal wells - All 24 bases were called and numQGtTwo>=12, were independent of their location with the tiles, " +
+            "and were directly proportional to the library concentration.  These reads are likely the result of PCR artifacts </li>" +
+            "<li>UNKNOWN - The remaining reads that PF-Failing but did not fit into any of the groups listed above</li>"   +
+            "</ul></p>  "+
+            "" +
+            "<p>The tool defaults to the SUMMARY output which indicates the number of PF-Failed reads per tile and groups " +
+            "them into the categories described above accordingly.</p> " +
+            "<p>A DETAILED metrics option is also available that subdivides the SUMMARY outputs by the x- y- position of these reads" +
+            " within each tile.  To obtain the DETAILED metric table, you must add the PROB_EXPLICIT_READS option and set the value " +
+            "between 0 and 1.  This value represents the fractional probability of PF-Failed reads to send to output.  " +
+            "For example, if PROB_EXPLICIT_READS=0, then no metrics will be output.  If PROB_EXPLICIT_READS=1, then it will " +
+            "provide detailed metrics for all (100%) of the reads.  It follows that setting the PROB_EXPLICIT_READS=0.5, " +
+            "will provide detailed metrics for half of the PF-Failed reads.</p> "+
+
+            "<p>Note: Metrics labeled as percentages are actually expressed as fractions!</p>" +
+            "" +
+            "<h4>Usage example: (SUMMARY Metrics)</h4> " +
+            "<pre>" +
+            "java -jar picard.jar CollectHiSeqXPfFailMetrics \\<br />" +
+            "      BASECALLS_DIR=/BaseCalls/ \\<br />" +
+            "      OUTPUT=/metrics/ \\<br />" +
+            "      LANE=001" +
+            "" +
+            "<h4>Usage example: (DETAILED)</h4>" +
+            "java -jar picard.jar CollectHiSeqXPfFailMetrics \\<br />" +
+            "      BASECALLS_DIR=/BaseCalls/ \\<br />" +
+            "      OUTPUT=/HiSeqX_pffail_summary_metrics.txt \\<br />" +
+            "      LANE=001 \\<br />" +
+            "      PROB_EXPLICIT_READS=1" +
+            "</pre>" +
```

Comment:
Glad you caught this.  I tried doing an ant build but it did not work.  So I could not see what this would look like.  Do I need to do something like "ant clean" first?